app/components/App.js

```diff
@@ -1,25 +1,27 @@
-var React = require('react');
+import React, {
+  Component
+} from 'react';
 
-var App = React.createClass({
-  render: function() {
-    var mainStyle = {
+export default class App extends Component {
```

Comment:
Do we need to render { this.props.children } in the app component in order to render nested routes?